observability/monitoring/configuring-the-monitoring-stack.adoc

```diff
@@ -211,6 +211,9 @@ ifndef::openshift-dedicated,openshift-rosa[]
 * For information about how to get your cluster ID, see xref:../../support/gathering-cluster-data.adoc#support-get-cluster-id_gathering-cluster-data[Obtaining your cluster ID].
 endif::openshift-dedicated,openshift-rosa[]
 
+// Configuring audit logs for Metrics Server
+include::modules/monitoring-configuring-audit-logs-for-metrics-server.adoc[leveloffset=+1]
```

Comment:
This module is also included in the OSD and ROSA distributions. Is that intended?

My initial hunch was that this shouldn't be in the OSD and ROSA docs, because the procedure involves changing a config map in the `openshift-monitoring` namespace. AFAIK, OSD and ROSA customers shouldn't have access to that namespace or change anything in it. It might be good to check with your SMEs about that.

Reply:
Hello @bhardesty I see, that is weird because I see the procedures in the ROSA and OSD distributions that include modifying the config map in the `openshift-monitoring` namespace :thinking: 
For example here:
ROSA:
* https://docs.openshift.com/rosa/observability/monitoring/configuring-the-monitoring-stack.html#specifying-limits-and-resource-requests-for-monitoring-components_configuring-the-monitoring-stack
* https://docs.openshift.com/rosa/observability/monitoring/configuring-the-monitoring-stack.html#managing-cpu-and-memory-resources-for-monitoring-components

OSD:
* https://docs.openshift.com/dedicated/observability/monitoring/configuring-the-monitoring-stack.html#managing-cpu-and-memory-resources-for-monitoring-components
* https://docs.openshift.com/dedicated/observability/monitoring/configuring-the-monitoring-stack.html#managing-cpu-and-memory-resources-for-monitoring-components

I will check if that is an issue in the ROSA and OSD distros.
Also, I am not sure if I should be the one to decide what belongs there, I think the ROSA and OSD teams make these changes as needed for the distros, but I will definitely check with @bburt-rh about this and let you know.

Thank you for making me aware of this!

Reply:
Okay so we were discussing what to do here, for now I will exclude it from the ROSA and OSD distros as that is a safer option for now. If it needs to be included in those distros, additional Jira will be raised.